C-Plus-Plus/cryptography/Homophonic_Cipher.cpp

```diff
@@ -0,0 +1,90 @@
+//Homophonic Subsitution Cipher
+
+/*
+
+The Homophonic Substitution cipher is a substitution cipher in which single 
+plaintext letters can be replaced by any of several different ciphertext letters. 
+They are generally much more difficult to break than standard substitution ciphers.
+
+A B C D E F G H I J K L M N O P Q R S T U V W X Y Z
+Q W E R T Y U I O P A S D F G H J K L Z X C V B N M
+4       5       8         7 2       1 6            
+        9                                          
+        0                         
+		
+		
+Example:
+Plain Text : TESSERACTCODING
+Cipher Text: 60LL5KQEZE2R87U
+
+Plain Text : NeoAlgo
+Cipher Text: 70gQsug
+
+Time Complexity: O(n)
+Space Complexity: O(1)		                 
+                   
+*/
+
+#include<bits/stdc++.h>
+using namespace std;
+
+void mapping(vector<vector<char> > &code){    //Mapping one plain-text alphabet to more than one cipher-text alphabet.
+	code[0].push_back('Q');code[0].push_back('4');
```

Comment:
Can you reformat this Code to prevent having so many redundant lines of Code?

Reply:
> Can you reformat this Code to prevent having so many redundant lines of Code?

Actually, that mapping is to be added manually. Therefore, I didn't find any other way to do it. If you could, kindly provide some way to reformat it.